ed/idlpatches/webcodecs.idl.patch

```diff
@@ -0,0 +1,96 @@
+From 3cc0e244620f53f827d50d90a07068bbfd038267 Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Mon, 14 Jun 2021 21:35:03 +0200
+Subject: [PATCH] Revert IDL while dictionary as attribute issue gets fixed
+
+https://github.com/w3c/webcodecs/issues/278
+---
+ ed/idl/webcodecs.idl | 46 +++++++++++++++++++-------------------------
+ 1 file changed, 20 insertions(+), 26 deletions(-)
+
+diff --git a/ed/idl/webcodecs.idl b/ed/idl/webcodecs.idl
+index dbb3cc2de..4fc4f975f 100644
+--- a/ed/idl/webcodecs.idl
++++ b/ed/idl/webcodecs.idl
+@@ -275,23 +275,22 @@ enum AudioSampleFormat {
+ [Exposed=(Window,DedicatedWorker)]
+ interface VideoFrame {
+   constructor(CanvasImageSource image, optional VideoFrameInit init = {});
+-  constructor(sequence<PlaneInit> planes,
```

Comment:
my previous comment applies also to this and other bits below - do we revert only the problematic bits (dictionaries used as attributes) or all the changes that were brought in at the same time?

Reply:
My worry is that the other updates are more intertwined and were done at the same time. I don't know whether the IDL makes a lot of sense if you only apply part of the changes. I would rather err on the safe side and freeze the IDL to what it was before.

Reply:
Thanks - that makes sense to me